other/Dockerfile

```diff
@@ -37,7 +40,11 @@ ADD prisma .
 RUN npx prisma generate
 
 ADD . .
-RUN npm run build
+
+# Mount the secret and set it as an environment variable and run the build
+RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
+    export SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && \
+    npm run build
```

Comment:
This is better 👍

Can you confirm this works if you don't have a `SENTRY_AUTH_TOKEN` when you run the docker build?

Reply:
cat command will fail if there is no secret but the exit code will be ignored because it is executed in a subprocess
<img width="638" alt="image" src="https://github.com/epicweb-dev/epic-stack/assets/702227/0cd235ae-372f-4b78-9710-33b2514cd5de">
also the error is written on stderr, it is not set on the variable
<img width="272" alt="image" src="https://github.com/epicweb-dev/epic-stack/assets/702227/5aa8b1ad-9f58-4705-be99-a105309e7ae5">

Reply:
Confirmed, yep, I removed the build secret from the `deploy.yml` and my app still built and deployed (but failed to send the source maps without the auth token, as you'd expect)

Reply:
Confirmed, I removed the secret from github for my own project and deploy succeeded fine, obviously without sending source maps due to missing auth token